src/get-queries-for-element.js

```diff
@@ -1,7 +1,22 @@
-import * as queries from './queries'
+import * as defaultQueries from './queries'
 
-function getQueriesForElement(element) {
-  return Object.entries(queries).reduce((helpers, [key, fn]) => {
+/**
+ * @typedef {{[key: string]: Function}} FuncMap
+ */
+
+/**
+ * @param {HTMLElement} element container
+ * @param {FuncMap|FuncMap[]} queries object of functions, or array of objects
+ * @returns {FuncMap} returns object of functions bound to container
+ */
+function getQueriesForElement(element, queries = defaultQueries) {
+  let flattenedQueries
+  if (Array.isArray(queries)) {
```

Comment:
I'd rather not have this logic here. If someone wants to merge the built in queries they can be responsible for doing that themselves when they call the function.